### ISSUE TITLE
[Feature] data-merchant-id support

### ISSUE THREAD

**Issue by beesheer** [paypal/paypal-js#60]
### 🚀 Feature Proposal

Could you add `data-merchant-id` support?

### Motivation

Without this script option support, we couldn't utilize to create multi seller order.
Direct js inclusion looks like this
```js
<script src="https://www.paypal.com/sdk/js?client-id=Client-ID&merchant-id=*" data-merchant-id="MERCHANT_ID_1,MERCHANT_ID_2"></script>
```

### Example

```js
loadScript({ 'client-id': 'YOUR_CLIENT_ID', 'merchant-id': '*', 'data-merchant-id': 'MERCHANT_ID_1,MERCHANT_ID_2' });
```

**Comment by gregjopa** [comment]
Hi @beesheer, thanks for reporting this issue. I added "data-merchant-id" to the script options type. This change is included in the paypal-js@2.1.7 release.

**Comment by beesheer** [comment]
Hey @gregjopa, that was quick for a weekend feature request! Appreciate it!
